docs/API.md

```diff
@@ -32,7 +32,7 @@ source = <<-EOS
 EOS
 
 reporter = Reek::Report::TextReport.new
-examiner = Reek::Examiner.new source
+examiner = Reek::Examiner.run(source)
```

Comment:
I'm not finding it very intuitive that `Reek::Examiner.run` or even `#run` should return an examiner. Why not write this out like so:

```
examiner = Reek::Examiner.new(source)
examiner.run
```

It's a little more verbose, but I think it's clearer.